server/routes.js

```diff
@@ -21,28 +18,69 @@ module.exports = [{
   config: {
     auth: {
       strategy: 'session',
+      // 'try': allow users to visit the route with good, bad, or no session
       mode: 'try'
     },
     plugins: {
       'hapi-auth-cookie': {
-        redirectTo: false
+        redirectTo: false // don't redirect users who don't have a session
       }
     },
     handler: function (request, reply) {
-      var page = request.auth.isAuthenticated ? 'app.html' : 'index.html';
-      // TODO we should set a session cookie here that's visible to the client: #45
+      // TODO: have a separate logged-in view :-)
+      // var page = request.auth.isAuthenticated ? 'app.html' : 'index.html';
+      var page = 'index.html';
       reply.file(path.join(STATIC_PATH, page));
     }
   }
 }, {
   method: 'GET',
-  path: '/auth/logout',
-  handler: function (request, reply) {
-    request.auth.session.clear();
-    return reply.redirect('/');
+  path: '/auth/login',
```

Comment:
I was thinking the same thing, but never got around to it. I'll make that change.